opendebates/templates/opendebates/changelog.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+<div class="row breadcrumbs">
+  <a href="/">{% blocktrans %}Home{% endblocktrans %}</a>
+  <span class="hidden-xs">&#8594;</span>
+  <a class="hidden-xs" href="">{% blocktrans %}Changelog{% endblocktrans %}</a>
+</div>
+<div>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Question</th>
+          <th>Date &amp; Time</th>
+          <th>Change Made</th>
+          <th>Information</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for idea in moderated %}
+        <tr>
+          <td>{{ idea.id }}</td>
+          <td><a href="{{ idea.get_absolute_url }}">{{ idea.headline }}</a></td>
+          <td>{{ idea.moderated_at|default_if_none:"--" }}</td>
+          <td>{% if idea.duplicate_of %}Merged{% else %}Removed{% endif %}</td>
+          <td>{% if idea.duplicate_of %}Merged into: <a href="{{ idea.duplicate_of.get_absolute_url }}">{{ idea.duplicate_of.headline }}</a>{% else %}--{% endif %}</td>
```

Comment:
Same comment here re three branches - if we want to be consistent with the rest of how the codebase talks about this, we'd distinguish "Merged into: XX" vs "Duplicate of: XX"